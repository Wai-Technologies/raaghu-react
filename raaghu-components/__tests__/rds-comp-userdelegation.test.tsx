import React from "react";
import RdsUserDelegations from "../src/rds-comp-user-delegations/rds-comp-user-delegations";


describe("RdsUserDelegations", () => {
    it("should render the component with the correct props", () => {
        const props = {
            onSubmit: jest.fn(),
            selectuser: [
                {
                    id: 1,
                    name: "John Doe",
                },
                {
                    id: 2,
                    name: "Jane Doe",
                },
            ],
        };
        const component = <RdsUserDelegations {...props} />;
        expect(component).toMatchSnapshot();
    });
});
