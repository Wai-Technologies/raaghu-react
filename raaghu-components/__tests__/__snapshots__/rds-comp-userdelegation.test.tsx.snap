// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RdsUserDelegations should render the component with the correct props 1`] = `
<RdsUserDelegations
  onSubmit={[MockFunction]}
  selectuser={
    [
      {
        "id": 1,
        "name": "John Doe",
      },
      {
        "id": 2,
        "name": "Jane Doe",
      },
    ]
  }
/>
`;
